gridappsd-field-bus-lib/gridappsd/field_interface/agents/agents.py

```diff
@@ -87,7 +91,7 @@ def __init__(self,
         #    self.addressable_equipments = agent_dict['addressable_equipment']
         #    self.unaddressable_equipments = agent_dict['unaddressable_equipment']
 
-    def connect(self):
+    def _connect(self):
```

Comment:
Why are we using _ here?

Reply:
It is used internally within this class while initialization. Not called by any agent.